tests/python/YapfBearTest.py

```diff
@@ -22,6 +24,15 @@ def test_valid(self):
                             ["x = {  'a':37,'b':42,\n", "'c':927}\n", '\n',
                              "y = 'hello ''world'\n"], valid=False)
 
+    def test_valid_python_2(self):
+        self.check_validity(self.uut, ['print 1\n'], valid=True)
```

Comment:
@AbdealiJK I think it's because of this special handling of the old print syntax in python 3:

```
>>> print 1
  File "<stdin>", line 1
    print 1
          ^
SyntaxError: Missing parentheses in call to 'print'
```

If Python 3 is able to recognize that the syntax is old, then the `ast` module **must** be able to parse this syntax. If the `ast` module is able to parse it, I guess it shouldn't fail.